frontend/src/app/spot/components/toggle/toggle.component.ts

```diff
@@ -36,8 +37,13 @@ export class SpotToggleComponent<T> implements ControlValueAccessor {
 
   @Input() public value:T;
 
+  constructor(
+    private cdRef:ChangeDetectorRef,
+  ) {}
+
   writeValue(value:T):void {
     this.value = value;
+    this.cdRef.markForCheck();
```

Comment:
This fixes the CVA failing to be used in OnPush situations. It requires to be marked for checked unless an input binding changes in the template, which is not the case here.